jip/main.py

```diff
@@ -20,14 +20,16 @@
 # SOFTWARE.
 #
 
-
+import os
 import sys
 import argparse
 
 from jip import logger
 from jip.commands import commands
 
+
 def main():
+    os.system("color")
```

Comment:
Will this affect execution on Linux or Mac?

Reply:
> 
> 
> Will this affect execution on Linux or Mac?

I checked and is causes a failure message when running 'color'.  I have change the logic to only execute if the running on Windows and using the CMD shell, PowerShell should work like Linux and Mac.